gitlab_runner/metadata.csv

```diff
@@ -1,50 +1,50 @@
-gitlab_runner.ci_runner_errors,count,,request,second,The number of caught errors. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,errors
-gitlab_runner.ci_runner_version_info,gauge,,request,,A metric with a constant '1' value labeled by different build stats fields. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,version info
-gitlab_runner.ci_ssh_docker_machines_provider_machine_creation_duration_seconds_bucket,gauge,,request,second,Histogram of SSH Docker machine creation time. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,ssh docker machine creation time
-gitlab_runner.ci_ssh_docker_machines_provider_machine_creation_duration_seconds_sum,gauge,,request,second,Sum of SSH Docker machine creation time. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,sum ssh docker machine creation time
-gitlab_runner.ci_ssh_docker_machines_provider_machine_creation_duration_seconds_count,gauge,,request,second,Count of SSH Docker machine creation time. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,count ssh docker machine creation time
-gitlab_runner.ci_ssh_docker_machines_provider_machine_states,gauge,,request,second,The current number of machines per state in this provider. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,total ssh docker machines per state
-gitlab_runner.gitlab_runner_autoscaling_machine_creation_duration_seconds,gauge,,request,second,Histogram of Docker machine creation time. Applies to GitLab Runner 1.11.0+,0,gitlab_runner,docker machine creation time
-gitlab_runner.gitlab_runner_autoscaling_machine_states,gauge,,request,second,The current number of machines per state in this provider. Applies to GitLab Runner 1.11.0+,0,gitlab_runner,total docker machines per state
+gitlab_runner.ci_docker_machines_provider_machine_creation_duration_seconds_bucket,gauge,,request,second,A histogram of Docker machine creation time. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,docker machine creation time
+gitlab_runner.ci_docker_machines_provider_machine_creation_duration_seconds_sum,gauge,,request,second,The sum of Docker machine creation time. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,sum docker machine creation time
+gitlab_runner.ci_docker_machines_provider_machine_creation_duration_seconds_count,gauge,,request,second,The count of Docker machine creation time. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,count docker machine creation time
+gitlab_runner.ci_docker_machines_provider_machine_states,gauge,,request,second,The current number of CI machines per state in this provider. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,total docker machines per state
+gitlab_runner.ci_runner_builds,gauge,,,,The current number of running builds. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,ci current running builds
+gitlab_runner.ci_runner_errors,gauge,,request,second,The number of caught errors. Applies to GitLab Runner < 1.11.0,0,gitlab_runner,ci errors
```

Comment:
Should this be kept as count since the metric is a prometheus counter?
```
# HELP ci_runner_errors The  number of catched errors.
# TYPE ci_runner_errors counter
ci_runner_errors{level="error"} 0
ci_runner_errors{level="fatal"} 0
ci_runner_errors{level="panic"} 0
ci_runner_errors{level="warning"} 0
```

Reply:
I changed it because it was marked as an incorrect type when using:     `aggregator.assert_metrics_using_metadata(get_metadata_metrics())`
I could change it back and enable the option to remove the type checks when calling the method

Reply:
We weren't setting `'send_monotonic_counter': True` in our e2e tests, which I think was the issue. I reverted the metric type change. Another option would be to just enable the option to remove the metric submission type check when calling the assert metadata

Reply:
No worries, the `gauge` is technically what is being submitted by default.